lib/Serialization/Deserialization.cpp

```diff
@@ -4473,6 +4473,8 @@ Expected<Type> ModuleFile::getTypeChecked(TypeID TID) {
   return typeOrOffset;
 }
 
+static Decl *handleErrorAndSupplyMissingMember(ASTContext& context, Decl *container, llvm::Error &&error);
```

Comment:
Nitpicks, although clang-format caught them:
- `&` goes on the name, not the base type.
- Please wrap to 80 columns.